Source/WebCore/platform/graphics/TextRun.h

```diff
@@ -182,6 +187,8 @@ class TextRun final : public CanMakeCheckedPtr<TextRun> {
     unsigned m_directionalOverride : 1; // Was this direction set by an override character.
     unsigned m_characterScanForCodePath : 1;
     unsigned m_disableSpacing : 1;
+
+    float m_effectiveZoom { 1.0f };
```

Comment:
I'm not sure that `TextRun` is the best place to propagate css zoom information. It should probably be set to FontCascade/FontCascadeDescription if needed.